CHANGELOG.md

```diff
@@ -1,5 +1,90 @@
 # Changelog
 
+## 4.7.0 (TBD)
+
+* Update bugsnag-android to v4.22.2:
+
+  * This release adds a compile-time dependency on the Kotlin standard library. This should not affect
+  the use of any API supplied by bugsnag-unity.
+
+  * Modularise bugsnag-android into Core, NDK, and ANR artifacts
+    [#522](https://github.com/bugsnag/bugsnag-android/pull/522)
+
+  * Migrate dependencies to androidx
+    [#554](https://github.com/bugsnag/bugsnag-android/pull/554)
+
+  * Report internal SDK errors to bugsnag
+    [#570](https://github.com/bugsnag/bugsnag-android/pull/570)
+    [#581](https://github.com/bugsnag/bugsnag-android/pull/581)
+    [#594](https://github.com/bugsnag/bugsnag-android/pull/594)
+    [#605](https://github.com/bugsnag/bugsnag-android/pull/605)
+    [#588](https://github.com/bugsnag/bugsnag-android/pull/588)
+    [#612](https://github.com/bugsnag/bugsnag-android/pull/612)
+
+
+  * Add `detectNdkCrashes` configuration option to bugsnag-android to toggle whether C/C++ crashes
+    are detected
+    [#491](https://github.com/bugsnag/bugsnag-android/pull/491)
+
+  * Use NetworkCallback to monitor connectivity changes on newer API levels
+  [#501](https://github.com/bugsnag/bugsnag-android/pull/501)
+
+  * Fix deserialization of custom stackframe fields in cached error reports
+    [#576](https://github.com/bugsnag/bugsnag-android/pull/576)
+
+  * Fix potential null pointer exception if `setMetaData` is called with a null
+    value
+
+  * Delete empty files left in cache directory, preventing an erroneous source of minimal error reports
```

Comment:
Isn't this only relevant because minimal reports existed? This entry probably isn't needed here.